astropy/visualization/time.py

```diff
@@ -231,11 +231,6 @@ def convert(self, value, unit, axis):
             """
             Convert a Time value to a scalar or array.
             """
-
-            # For Matplotlib < 2.2
-            if not isinstance(value, Time):
```

Comment:
We don't  need this anymore, right, @astrofrog or @mhvk ?

Reply:
I think you're right - and will feel confident if tests pass!